### ISSUE TITLE
Update for latest DUO app.

### ISSUE THREAD

**Issue by Haldyz** [adishy/AutomateDuo#3]
Hello,

its already some time, since DUO app UI was updated and this script stopped working.
Please may you provide an update or guide me what needs to be updated to catch notification from new GUI?

Thanks in advance

**Comment by opmetal** [comment]
Seconded. The DUO UI update breaks this process. It would be great if you could provide an update or some guidance on how to fix it. Thank you!

**Comment by adamllryan** [comment]
If you guys still need an answer, all you need to do is set up the script normally, edit task T, open the action _AutoNotification Actions_, edit configuration, and change %anbutton1action to %anbutton2action. This should fix the script. I can link the XML file if you'd like.

**Comment by opmetal** [comment]
Hey adamllryan, if you could link the XML that would help out a lot. I'm still having trouble getting this to work.

**Comment by adamllryan** [comment]
I tried uploading it in this comment, but Github says .XML is unsupported, so I created a fork and replaced the old XML file. 
https://github.com/adamllryan/AutomateDuo
It has the same name and the instructions should all be the same. 

There is still the issue of the task failing when the screen is off that I'm still trying to fix, I will let you know if I fix it.

**Comment by opmetal** [comment]
Thanks for your help on this. It works but only if I run the task manually. I have to first tap the initial DUO popup at the top of the screen, which then opens up the accept/deny dialog. At this point I can run the task and it works.

**Comment by Haldyz** [comment]
Hi, thx for update, I have changed %anbutton**1**action to %anbutton2action and its finally working.
Actually for me its working even when the screen is off. But I have a different definition of the tasker Profile - I'm not using Event type "Notification", but Event type "Plugin" where I have selected "AutoNotification" and then I have selected "Intercept" Event Behavior - i have attached xmls for my Profile even the Task
[DUO_profile+task.zip](https://github.com/adishy/AutomateDuo/files/8206994/DUO_profile%2Btask.zip)
![DUO Tasker profile](https://user-images.githubusercontent.com/77940005/157258342-8589c0ec-8497-4021-8335-fff288c9a729.jpg)

**Comment by adamllryan** [comment]
That's interesting, I tried your setup and it did not work for me. Do you have a password on your phone? It looks like tasker is idling until I unlock my phone.

**Comment by Haldyz** [comment]
Yes, I'm using fingerprint / gesture.

**Comment by adamllryan** [comment]
Yeah, this is a little weird. It works when my screen is on but not logged in. That's alright though, works well enough for me. Glad the task works now!

**Comment by alexlyzhov** [comment]
@Haldyz thank you, this worked! Haven't tried it with screen off yet.

**Comment by balakumarane** [comment]
Is this method still working ?

**Comment by opmetal** [comment]
> Is this method still working ?

Yep!
